bot/exts/info/information.py

```diff
@@ -533,21 +554,30 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
             await ctx.send(embed=rules_embed)
             return
 
-        full_rules = await self.bot.api_client.get("rules", params={"link_format": "md"})
-
         # Remove duplicates and sort the rule indices
-        rules = sorted(set(rules))
+        rule_numbers = sorted(set(rule_numbers))
 
-        invalid = ", ".join(str(index) for index in rules if index < 1 or index > len(full_rules))
+        invalid = ", ".join(
+            str(rule_number) for rule_number in rule_numbers
+            if rule_number < 1 or rule_number > len(full_rules))
 
         if invalid:
             await ctx.send(shorten(":x: Invalid rule indices: " + invalid, 75, placeholder=" ..."))
             return
 
-        for rule in rules:
-            self.bot.stats.incr(f"rule_uses.{rule}")
+        final_rule_numbers = []
+        final_rule_numbers.extend(rule_numbers)
+
+        for keyword in keywords:
+            final_rule_numbers.append(keyword_to_rule_number.get(keyword))
```

Comment:
Avoid using `get()` if you don't need to rely on its default value. Otherwise, it may mislead readers into thinking that sometimes the key may not exist. Furthermore, the code below assumes everything in `final_rule_numbers` is a valid index. If a bug was somehow introduced that caused a key to not exist in `keyword_to_rule_number`, the resulting error would be clearer if it originated from this point than from somewhere below.